style.css

```diff
@@ -0,0 +1,577 @@
+* {
+  margin: 0;
```

Comment:
This is just a personal preference of mine - but I don't like setting `margin:0` using the the `*` selector, because it'll remove the margin from every element on the page. A lot of elements have margin by default, so this means you would have to re-add the margin for those elements.